Sources/CGImage.swift

```diff
@@ -50,10 +50,13 @@ public class CGImage {
 
     internal convenience init?(_ sourceData: Data) {
         var data = sourceData
-        let dataCount = Int32(data.count)
 
-        guard let gpuImagePtr = data.withUnsafeMutableBytes({ (ptr: UnsafeMutablePointer<Int8>) -> UnsafeMutablePointer<GPU_Image>? in
-            let rw = SDL_RWFromMem(ptr, dataCount)
+        guard let gpuImagePtr = data.withUnsafeMutableBytes({ buffer -> UnsafeMutablePointer<GPU_Image>? in
```

Comment:
hmm interesting...buffer is now an `UnsafeBufferPointer` right? So previously we used it as an `UnsafeMutablePointer<Int8>` which was not so correct?

Reply:
The old API is deprecated – the new code essentially does the same thing though. I just took the opportunity to get rid of some build warnings while cleaning up the other bits